### PR TITLE
Tune links in tasks section (2/2)

### PR DESCRIPTION
**Pull request by tengqm**

### CODE REVIEW

File: content/en/docs/tasks/access-application-cluster/access-cluster.md [hunk 1]
```diff
@@ -29,8 +26,9 @@ Check the location and credentials that kubectl knows about with this command:
 kubectl config view
 ```
 
-Many of the [examples](/docs/user-guide/kubectl-cheatsheet) provide an introduction to using
-kubectl and complete documentation is found in the [kubectl manual](/docs/user-guide/kubectl-overview).
+Many of the [examples](/docs/reference/kubectl/cheatsheet) provide an introduction to using
+kubectl and complete documentation is found in the
```
Comment (kbhawkey) [inline content/en/docs/tasks/access-application-cluster/access-cluster.md:30]: nits: /docs/reference/kubectl/cheatsheet/, /docs/reference/kubectl/overview/

**Comment by bboreham** [comment]
Can you give some context for this change please?
Are the old links broken?

File: content/en/docs/tasks/access-application-cluster/connecting-frontend-backend.md [hunk 2]
```diff
@@ -11,34 +11,23 @@ microservice. The backend microservice is a hello greeter. The
...
   backend application.
 
-
-
-
 ## {{% heading "prerequisites" %}}
 
-
-* {{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
```
Comment (kbhawkey) [inline content/en/docs/tasks/access-application-cluster/connecting-frontend-backend.md:23]: nit: line 25, Do you need a list at line 25?

File: content/en/docs/tasks/administer-cluster/cluster-management.md [hunk 3]
```diff
@@ -85,28 +82,37 @@ To upgrade a cluster on a platform not mentioned in the above list, check the or
 
 ## Resizing a cluster
 
-If your cluster runs short on resources you can easily add more machines to it if your cluster is running in [Node self-registration mode](/docs/admin/node/#self-registration-of-nodes).
-If you're using GCE or Google Kubernetes Engine it's done by resizing the Instance Group managing your Nodes. It can be accomplished by modifying number of instances on `Compute > Compute Engine > Instance groups > your group > Edit group` [Google Cloud Console page](https://console.developers.google.com) or using gcloud CLI:
+If your cluster runs short on resources you can easily add more machines to it if your cluster
+is running in [Node self-registration mode](/docs/admin/node/#self-registration-of-nodes).
```
Comment (kbhawkey) [inline content/en/docs/tasks/administer-cluster/cluster-management.md:86]: nit: Fix `/docs/admin/node/`?

File: content/en/docs/tasks/administer-cluster/network-policy-provider/romana-network-policy.md [hunk 4]
```diff
@@ -10,14 +10,9 @@ weight: 40
...
 
-
-
 ## {{% heading "prerequisites" %}}
 
-
-Complete steps 1, 2, and 3 of the [kubeadm getting started guide](/docs/getting-started-guides/kubeadm/).
-
-
+Complete steps 1, 2, and 3 of the [kubeadm getting started guide](/docs/reference/setup-tools/kubeadm/kubeadm).
```
Comment (kbhawkey) [inline content/en/docs/tasks/administer-cluster/network-policy-provider/romana-network-policy.md:15]: nit:/docs/reference/setup-tools/kubeadm/kubeadm/

File: content/en/docs/tasks/administer-cluster/network-policy-provider/weave-network-policy.md [hunk 5]
```diff
@@ -10,22 +10,21 @@ weight: 50
...
-
-
 ## {{% heading "prerequisites" %}}
 
-
-You need to have a Kubernetes cluster. Follow the [kubeadm getting started guide](/docs/getting-started-guides/kubeadm/) to bootstrap one.
-
-
+You need to have a Kubernetes cluster. Follow the
+[kubeadm getting started guide]( /docs/reference/setup-tools/kubeadm/kubeadm/) to bootstrap one.
```
Comment (kbhawkey) [inline content/en/docs/tasks/administer-cluster/network-policy-provider/weave-network-policy.md:16]: nit: extra space before `/docs/...`

**Comment by kbhawkey** [comment]
Previews:
https://deploy-preview-23017--kubernetes-io-master-staging.netlify.app/docs/tasks/tls/manual-rotation-of-ca-certificates/

**Comment by kbhawkey** [comment]
@tengqm , I noted several nits/questions, otherwise looks good.

File: content/en/docs/tasks/access-application-cluster/connecting-frontend-backend.md (hunk 2 above)
Comment (tengqm) [reply content/en/docs/tasks/access-application-cluster/connecting-frontend-backend.md:23]: rechecked the shortcode ...
you are right, this shortcode is meant to be used as a standalone paragraph and it already contains a list.

File: content/en/docs/tasks/administer-cluster/cluster-management.md (hunk 3 above)
Comment (tengqm) [reply content/en/docs/tasks/administer-cluster/cluster-management.md:86]: done.

**Comment by tengqm** [comment]
@bboreham There ARE missing links in the tasks section, due to incomplete refactoring or revision or whatever reasons. The PR is mainly fixing links that are using redirects which make the links difficult to maintain.

**Comment by kbhawkey** [comment]
@tengqm , Thanks for cleaning up the links and spacing.
/lgtm

**Pull request merged**